### PR TITLE
Cleanup ShaderNode

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: -

**Description**

Cleanup ShaderNode.

### CODE REVIEW

**Comment by LeviPesin** [comment]
> Let's avoid declaring ShaderNode outside of ShaderNode.js

Why? I think it would be better not to export NodeHandler and ShaderNodeScript directly.

> float, uint, etc outside of ShaderNodeElements.

I did not revert this change here (I only made a `cacheMaps` object from cacheMaps).

File: examples/jsm/nodes/shadernode/ShaderNode.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import { ShaderNodeScript, NodeHandler } from './ShaderNodeUtils.js';
-
-const ShaderNode = new Proxy( ShaderNodeScript, NodeHandler );
```
Comment (sunag) [inline examples/jsm/nodes/shadernode/ShaderNode.js:3]: @LeviPesin can you revert this part?

**Comment by sunag** [comment]
> Why? I think it would be better not to export NodeHandler and ShaderNodeScript directly.

It would be the correct representation of file and class.
There are also plans to turn `ShaderNode` into an extended `Node`.

> I did not revert this change here (I only made a cacheMaps object from cacheMaps).

Yes, yes!! So I reopened, that part was great. :)

**Comment by LeviPesin** [comment]
OK, is it better now?

File: examples/jsm/nodes/shadernode/ShaderNodeUtils.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import SplitNode from '../utils/SplitNode.js';
 import ConstNode from '../core/ConstNode.js';
 import { getValueFromType } from '../core/NodeUtils.js';
 
-const NodeHandler = {
+export const shaderNodeHandler = {
```
Comment (sunag) [inline examples/jsm/nodes/shadernode/ShaderNodeUtils.js:8]: `shaderNodeHandler` has a constructor, it can be `ShaderNodeHandler`

Comment (LeviPesin) [reply examples/jsm/nodes/shadernode/ShaderNodeUtils.js:8]: But it is just an object and not a constructor - you can theoretically use `new shaderNodeHandler.constructor()`, but not `new shaderNodeHandler()`.

**Pull request merged**